awx/main/scheduler/task_manager.py

```diff
@@ -492,8 +498,14 @@ def process_pending_tasks(self, pending_tasks):
         for task in pending_tasks:
             if self.start_task_limit <= 0:
                 break
-            if self.is_job_blocked(task):
+            blocked_by = self.job_blocked_by(task)
+            if blocked_by:
                 logger.debug("{} is blocked from running".format(task.log_format))
+                logger_job_lifecycle.info(f"{task._meta.model_name}-{task.id} blocked by {blocked_by._meta.model_name}-{blocked_by.id}", extra={'type': task._meta.model_name, 'template_name': task.unified_job_template.name, 'job_id': task.id, 'state': 'blocked', 'blocked_by': f"{blocked_by._meta.model_name}-{blocked_by.id}"})
+                job_explanation = str.format(f"waiting for {blocked_by._meta.model_name}-{blocked_by.id} to finish")
```

Comment:
Do we need the `str.format` since we're using an f-string?  Let's also wrap this string in i18n `_()` function call so it gets marked for translation.